pizza-time/src/components/friend-search-box/friend-search-box.js

```diff
@@ -1,13 +1,71 @@
 import React from "react";
+import axios from "axios";
+import NewFriend from "./NewFriend.js"
 
 //search box to find a new friend
 //used in: profile
-export default function FriendSearchBox() {
-  return (
-    <>
-      <form>
-        <input type="text" name="fname" placeholder="Find a new friend" />
-      </form>
-    </>
-  );
+// export default function FriendSearchBox() {
+//   return (
+//     <>
+//       <form>
+//         <input type="text" name="fname" placeholder="Find a new friend" />
+//       </form>
+//     </>
+//   );
+// }
+
+class FriendSearchBox extends React.Component{
+        constructor(props){
+          super(props);
+          this.state={
+            
+            searchName : "",
+            matchedUsers:[
+            ]
+          }
+        }
+        searchHandler=(event)=>{
+          event.preventDefault();
+            const name=this.state.searchName
+          if(name.length===0){
+            return
+          }
+          
+          
+         const id =localStorage.getItem("userFireBaseId")
+               
+          console.log("HERE IS THE ID",id)
+          let matchedUsers  = axios.post("http://localhost:5500/api/users/find/name/", {
```

Comment:
change this to use the deployed backend please. Everything should be running on that now.